hedera-node/hedera-app/src/main/java/module-info.java

```diff
@@ -21,6 +21,7 @@
     requires transitive com.swirlds.metrics.api;
     requires transitive com.swirlds.platform.core;
     requires transitive com.swirlds.state.api;
+    requires transitive com.swirlds.state.impl;
```

Comment:
Since it is transitive, and state.impl requires state.api, is state.api above redundant?

Reply:
Yes. I removed `requires transitive com.swirlds.state.api;`